tensorflow/tensorflow.bzl

```diff
@@ -202,7 +202,7 @@ def tf_copts(android_optimization_level_override="-O2", is_external=False):
           "-ftemplate-depth=900"])
       + if_cuda(["-DGOOGLE_CUDA=1"])
       + if_tensorrt(["-DGOOGLE_TENSORRT=1"])
-      + if_mkl(["-DINTEL_MKL=1", "-DEIGEN_USE_VML", "-fopenmp",])
+      + if_mkl(["-DINTEL_MKL=1", "-DEIGEN_USE_VML"])
```

Comment:
@mahmoud-abuzaina How will this effect Linux builds?

Reply:
We have some op kernels (e.g., Slice) optimized using OpenMP. This change will disable such optimizations unless `-fopenmp` gets passed some other way.

Reply:
Thanks for feedback @claynerobison and @nhasabni.
Adding `-fopenmp` under double condition of if_mkl and if_linux appears to be tricky as bazel doesn't support nested select https://github.com/bazelbuild/bazel/issues/1623
The only working solution I have found so far is to update `set_cc_opt_flags` in `configure.py` with these lines
```
  if is_linux():
    write_to_bazelrc("build:mkl --copt=-fopenmp")
    write_to_bazelrc("build:mkl --host_copt=-fopenmp")
```
Please let me know if this is acceptable solution or suggest any other options.

P.S.
I have never used/modified bazel before, so suggestions are very welcome.

Reply:
One possible way is creating `if_mkl_linux` and `if_mkl_macos` macros.
I have doodled around these a while ago here:
https://github.com/gunan/tensorflow/commit/b4d39c5226e8bbafe084b1dfc35f4644f2d906c2
Feel free to use the changes from there.

Reply:
Thanks! Let me try it out.

Reply:
That worked! Thanks @gunan! Now I know how to use `config_setting` =)